src/app/core/guards/home.guard.ts

```diff
@@ -0,0 +1,44 @@
+import { Injectable } from '@angular/core';
+import { ActivatedRouteSnapshot, CanActivate, Router, RouterStateSnapshot, UrlTree } from '@angular/router';
+import { Observable } from 'rxjs';
+import { LocalStorageService } from '@app-services';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class HomeGuard implements CanActivate {
+
+  constructor(private storageService: LocalStorageService, private router: Router) { }
+
+  canActivate(
+    route: ActivatedRouteSnapshot,
+    state: RouterStateSnapshot): Observable<boolean | UrlTree> | Promise<boolean | UrlTree> | boolean | UrlTree {
+    return this.checkLogin(state.url);
+  }
+
+  /**
+   * This method go to login page if user is logged
+   * 
+   * @param url 
+   * 
+   * @returns url or boolean
+   */
+  public checkLogin(url: string): any {
+    let storageLoggedIn: any = this.storageService.getItem('isUserLoggedIn');
+
+    if (url === "/login") {
```

Comment:
This if seems to complex, try to refactor or at least add comments or more variables that could more describe if.